ipn/ipnlocal/local_test.go

```diff
@@ -1885,6 +1901,47 @@ func TestSetExitNodeIDPolicy(t *testing.T) {
 	}
 }
 
+func TestUpdateNetmapDeltaAutoExitNode(t *testing.T) {
+	b := newTestLocalBackend(t)
+	policy := "auto"
+	msh := &mockSyspolicyHandler{
+		t: t,
+		stringPolicies: map[syspolicy.Key]*string{
+			syspolicy.ExitNodeID: &policy,
+		},
+	}
+	syspolicy.SetHandlerForTest(t, msh)
+	b.netMap = &netmap.NetworkMap{}
+	for i := range 2 {
+		b.netMap.Peers = append(b.netMap.Peers, (&tailcfg.Node{ID: (tailcfg.NodeID(i) + 1), StableID: tailcfg.StableNodeID(fmt.Sprint(i + 1)), Cap: 26}).View())
```

Comment:
Check out the helpers I made:
```golang
b.netMap.Peers = append(b.netMap.Peers, makePeer(tailcfg.NodeID(i+1), withCap(26)))
```

You'll have to create `withCap()` but it should be easy, look at `withName()` right below `makePeer()` in this same file.